### PR TITLE
Dependency bumps

### PR DESCRIPTION
**Pull request by yschimke**
Want to separate these dependency bumps from 

https://github.com/square/okhttp/pull/8600

Okio leading the push to 2.1.0 in https://github.com/square/okio/pull/1552

### CODE REVIEW

**Comment by yschimke** [comment]
Failing with

```
Caused by: org.junit.platform.commons.JUnitException: TestEngine with ID 'junit-jupiter' failed to discover tests
	at app//org.junit.platform.launcher.core.EngineDiscoveryOrchestrator.discoverEngineRoot(EngineDiscoveryOrchestrator.java:160)
	at app//org.junit.platform.launcher.core.EngineDiscoveryOrchestrator.discoverSafely(EngineDiscoveryOrchestrator.java:132)
	at app//org.junit.platform.launcher.core.EngineDiscoveryOrchestrator.discover(EngineDiscoveryOrchestrator.java:107)
	at app//org.junit.platform.launcher.core.EngineDiscoveryOrchestrator.discover(EngineDiscoveryOrchestrator.java:78)
	at app//org.junit.platform.launcher.core.DefaultLauncher.discover(DefaultLauncher.java:99)
	at app//org.junit.platform.launcher.core.DefaultLauncher.execute(DefaultLauncher.java:85)
	at app//org.junit.platform.launcher.core.DelegatingLauncher.execute(DelegatingLauncher.java:47)
	at org.gradle.api.internal.tasks.testing.junitplatform.JUnitPlatformTestClassProcessor$CollectAllTestClassesExecutor.processAllTestClasses(JUnitPlatformTestClassProcessor.java:124)
	at org.gradle.api.internal.tasks.testing.junitplatform.JUnitPlatformTestClassProcessor$CollectAllTestClassesExecutor.access$000(JUnitPlatformTestClassProcessor.java:99)
	at org.gradle.api.internal.tasks.testing.junitplatform.JUnitPlatformTestClassProcessor.stop(JUnitPlatformTestClassProcessor.java:94)
	at org.gradle.api.internal.tasks.testing.SuiteTestClassProcessor.stop(SuiteTestClassProcessor.java:63)
	... 18 more
Caused by: org.junit.platform.commons.JUnitException: ClassSelector [className = 'okhttp3.osgi.OsgiTest', classLoader = jdk.internal.loader.ClassLoaders$AppClassLoader@7aec35a] resolution failed
	at app//org.junit.platform.launcher.listeners.discovery.AbortOnFailureLauncherDiscoveryListener.selectorProcessed(AbortOnFailureLauncherDiscoveryListener.java:39)
	at app//org.junit.platform.engine.support.discovery.EngineDiscoveryRequestResolution.resolveCompletely(EngineDiscoveryRequestResolution.java:103)
	at app//org.junit.platform.engine.support.discovery.EngineDiscoveryRequestResolution.run(EngineDiscoveryRequestResolution.java:83)
	at app//org.junit.platform.engine.support.discovery.EngineDiscoveryRequestResolver.resolve(EngineDiscoveryRequestResolver.java:113)
	at app//org.junit.jupiter.engine.discovery.DiscoverySelectorResolver.resolveSelectors(DiscoverySelectorResolver.java:48)
	at app//org.junit.jupiter.engine.JupiterTestEngine.discover(JupiterTestEngine.java:69)
	at app//org.junit.platform.launcher.core.EngineDiscoveryOrchestrator.discoverEngineRoot(EngineDiscoveryOrchestrator.java:152)
	... 28 more
Caused by: java.lang.UnsupportedClassVersionError: aQute/bnd/build/Workspace has been compiled by a more recent version of the Java Runtime (class file version 61.0), this version of the Java Runtime only recognizes class file versions up to 55.0
	at java.base/java.lang.ClassLoader.defineClass1(Native Method)
	at java.base/java.lang.ClassLoader.defineClass(ClassLoader.java:1022)
```

File: gradle.properties [hunk 1]
```diff
@@ -18,11 +18,11 @@ containerTests=false
 # - CodeCache normally defaults to a very small size. Increasing it from platform defaults of 32-48m
 #   because of how many classes can be loaded into memory and then cached as native compiled code
 #   for a small speed boost.
-org.gradle.jvmargs=-Dfile.encoding=UTF-8 -XX:+UseG1GC -XX:SoftRefLRUPolicyMSPerMB=1 -XX:ReservedCodeCacheSize=256m -XX:+HeapDumpOnOutOfMemoryError -Xmx1g -Xms1g
+org.gradle.jvmargs=-Dfile.encoding=UTF-8 -XX:+UseG1GC -XX:SoftRefLRUPolicyMSPerMB=1 -XX:ReservedCodeCacheSize=256m -XX:+HeapDumpOnOutOfMemoryError -Xmx2g -Xms2g
```
Comment (swankjesse) [inline gradle.properties:21]: Any idea what is motivating this?

Comment (yschimke) [reply gradle.properties:21]: Sorry, it's meant to be 4gb. From the tuned project I cribbed from.

But I was getting failures I thought were memory related. And was testing.

I still don't think we can have any 4x4gb

**Comment by yschimke** [comment]
@swankjesse Going to land this to see if I can make forward progress on the JVM/Android split PR.  Happy to fix forward with any review feedback.

File: build.gradle.kts [hunk 2]
```diff
@@ -179,6 +181,10 @@ subprojects {
 
     maxParallelForks = Runtime.getRuntime().availableProcessors() * 2
     testLogging {
+      if (System.getenv("CI") != null) {
+          events = setOf(TestLogEvent.FAILED)
+          info.events = setOf(TestLogEvent.FAILED)
```
Comment (JakeWharton) [inline build.gradle.kts:186]: Why both?

**Review by JakeWharton — APPROVED** [review]

File: build.gradle.kts (hunk 2 above)
Comment (yschimke) [reply build.gradle.kts]: Reverted this part, I was trying to improve github CI output while struggling with it, but its a separate problem.

File: gradle.properties (hunk 1 above)
Comment (yschimke) [reply gradle.properties]: reverted from the PR

**Pull request merged**